dashboard/urls.py

```diff
@@ -34,3 +37,10 @@
     # django urls
     url(r'^admin/', admin.site.urls),
 ]
+
+# Make some error pages available under error/ for easier testing
```

Comment:
Hmm ... I can't promise I won't take this out later to simplify the code.

FWIW, I'm able to easily test by setting `DJANGO_DEBUG=True/False` in the `.env` file, and then running `source .env` before `runserver`.